src/features/lightspeed/lightSpeedOAuthProvider.ts

```diff
@@ -86,6 +89,11 @@ export class LightSpeedAuthenticationProvider
     );
   }
 
+  async setExternalRedirectUri() {
+    const callbackUri = await env.asExternalUri(Uri.parse(this.redirectUri));
```

Comment:
When VS Code is running as a standalone application, env.asExternalUri() will return the input URI without modifications ([reference](https://code.visualstudio.com/api/references/vscode-api#env.asExternalUri)).